internal/service/odb/cloud_autonomous_vm_cluster.go

```diff
@@ -0,0 +1,692 @@
+					float64planmodifier.RequiresReplace(),
+				},
+				Description: "The data storage size allocated for Autonomous Databases in the Autonomous VM cluster, in TB. Changing this will force terraform to create new resource.",
+			},
+			"available_autonomous_data_storage_size_in_tbs": schema.Float64Attribute{
+				Computed:    true,
+				Description: "The available data storage space for Autonomous Databases in the Autonomous VM cluster, in TB.",
+			},
+			"available_container_databases": schema.Int32Attribute{
+				Computed:    true,
+				Description: "The number of Autonomous CDBs that you can create with the currently available storage.",
+			},
+			"available_cpus": schema.Float32Attribute{
+				Computed:    true,
+				Description: "The number of CPU cores available for allocation to Autonomous Databases",
+			},
+			"compute_model": schema.StringAttribute{
+				CustomType:  computeModel,
+				Computed:    true,
+				Description: "The compute model of the Autonomous VM cluster: ECPU or OCPU.",
+			},
+			"cpu_core_count": schema.Int32Attribute{
+				Computed:    true,
+				Description: "The total number of CPU cores in the Autonomous VM cluster.",
+			},
+			"cpu_core_count_per_node": schema.Int32Attribute{
+				Required: true,
+				PlanModifiers: []planmodifier.Int32{
+					int32planmodifier.RequiresReplace(),
+				},
+				Description: "The number of CPU cores enabled per node in the Autonomous VM cluster.",
+			},
+			"cpu_percentage": schema.Float32Attribute{
+				Computed:    true,
+				Description: "The percentage of total CPU cores currently in use in the Autonomous VM cluster.",
+			},
+			names.AttrCreatedAt: schema.StringAttribute{
+				Computed:    true,
+				CustomType:  timetypes.RFC3339Type{},
+				Description: "The date and time when the Autonomous VM cluster was created.",
+			},
+			"data_storage_size_in_gbs": schema.Float64Attribute{
+				Computed:    true,
+				Description: "The total data storage allocated to the Autonomous VM cluster, in GB.",
+			},
+			"data_storage_size_in_tbs": schema.Float64Attribute{
+				Computed:    true,
+				Description: "The total data storage allocated to the Autonomous VM cluster, in TB.",
+			},
+			"odb_node_storage_size_in_gbs": schema.Int32Attribute{
+				Computed:    true,
+				Description: " The local node storage allocated to the Autonomous VM cluster, in gigabytes (GB)",
+			},
+			"db_servers": schema.SetAttribute{
+				Required:    true,
+				CustomType:  fwtypes.SetOfStringType,
+				ElementType: types.StringType,
+				PlanModifiers: []planmodifier.Set{
+					setplanmodifier.RequiresReplace(),
+				},
+				Description: "The database servers in the Autonomous VM cluster. Changing this will force terraform to create new resource.",
+			},
+			names.AttrDescription: schema.StringAttribute{
+				Optional: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+					stringplanmodifier.UseStateForUnknown(),
```

Comment:
`UseStateForUnknown` plan modifier should not be necessary here since this attribute is not computed